navbar-component.html

```diff
@@ -22,15 +24,16 @@
       </div>
 
       <!-- Collect the nav links, forms, and other content for toggling -->
-      <div class="collapse navbar-collapse" id="br-navbar-collapse">
+      <div ng-if="!$ctrl.service.brandOnly" class="collapse navbar-collapse"
+        id="br-navbar-collapse">
```

Comment:
General rule: don't ever use `id` attribute. In this case, use the `uib-collapse` angular component. For example:

https://github.com/digitalbazaar/authorization.io/blob/facelift/components/register-component.html#L26